packages/@ama-sdk/schematics/package.json

```diff
@@ -43,6 +43,7 @@
     "@angular-devkit/schematics": "~16.0.5",
     "@angular-devkit/schematics-cli": "~16.0.5",
     "@angular/cli": "~16.0.5",
+    "@o3r/schematics": "workspace:^",
```

Comment:
Should be optional.
It is used only for `ng-add` and it can make sense in a Otter project context but a bit less in an SDK.